ansible/playbook-docker.yml

```diff
@@ -5,13 +5,25 @@
   vars_files:
     - ./vars.yml
 
+  vars:
+    sql_files:
+      - { source: "pgbouncer_config/pgbouncer_auth_schema.sql", dest: "00-schema.sql" }
```

Comment:
For now, the docker images don't have PgBouncer attached to it. Would you still want this included to ensure as much parity as possible with the hosted image?

Reply:
yeah I think that's preferrable for now